Ginger/Ginger/Actions/_Common/ActUIElementLib/ActUIElementEditPage.xaml.cs

```diff
@@ -121,22 +148,29 @@ private void SetLocateValueFrame()
 
         private void ElementTypeComboBox_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            mElementActionsList = mPlatform.GetPlatformUIElementActionsList(mAction.ElementType);
+            if (Convert.ToBoolean(mAction.GetInputParamValue(Fields.IsWidgetsElement)))
+            {
+                mElementActionsList = mPlatform.GetPlatformWidgetsUIActionsList(mAction.ElementType);
+            }
+            else
+            {
+                mElementActionsList = mPlatform.GetPlatformUIElementActionsList(mAction.ElementType);
```

Comment:
put this part in if by checking is iswidget element ==false because most of the time it will be false